section1/exercises/strings.rb

```diff
@@ -7,9 +7,20 @@
 p "Alan Turing"
 
 # Write code that prints `Welcome to Turing!` to the terminal:
-p #YOUR CODE HERE
+<<<<<<< HEAD
+puts "Matt Holmes"
 
 # Write code that prints `99 bottles of pop on the wall...` to the terminal:
-# YOUR CODE HERE
+puts "99 bottles of pop on the wall..."
 
-# Write out code to log one line from your favorite song or movie.
\ No newline at end of file
+# Write out code to log one line from your favorite song or movie.
+puts "Waking up dead inside of my head"
+=======
+puts "Alan Turing"
+
+# Write code that prints `99 bottles of pop on the wall...` to the terminal:
+puts "99 bottles of pop on the wall... "
+
+# Write out code to log one line from your favorite song or movie.
+puts "Waking up dead inside of my head/Will never never do there is no med"
+>>>>>>> d969eee7c599dfbc09ec03745691ba18633c293c
```

Comment:
This line comes from a merge.. make sure you remove these otherwise your code does not run